Formula/dashlane-cli.rb

```diff
@@ -0,0 +1,37 @@
+require "language/node"
+
+class DashlaneCli < Formula
+  desc "Command-line interface for Dashlane"
+  homepage "https://dashlane.com"
+  url "https://github.com/Dashlane/dashlane-cli/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "6345319412b5d224083bc032f4a1ba1890ace5ecf70a63f4e9930aa6537d1d49"
+  license "Apache-2.0"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+    regex(%r{href=.*?/tag/vv?(\d+(?:\.\d+)+)["' >]}i)
+  end
+
+  depends_on "node@16"
+
+  on_linux do
+    depends_on "glibc"
+  end
+
+  def install
+    Language::Node.setup_npm_environment
+    platform = OS.linux? ? "linux" : "macos"
+    libc = OS.linux? ? "glibc" : "unknown"
+    arch = `uname -m`
+    system "npm", "ci"
+    system "npm", "run", "build"
+    system "./prepare-pkg.sh", libc
+    system "npx", "pkg", ".", "-t", "node16-#{platform}-#{arch.chomp}", "-o", "dcli"
+    bin.install "dcli"
+  end
+
+  test do
+    assert_equal version.to_s, shell_output("#{bin}/dcli --version").chomp
```

Comment:
Can you add a check that at least covers some of its functionality? Version checks like this almost never fail.

Reply:
All other commands requires inputs from users with an active account on Dashlane service, if you know a way to input secrets in the tests I'll be happy to add extra tests.

Reply:
How about checking if it fails correctly?

Reply:
> if you know a way to input secrets

That's not possible, unfortunately.